### PR TITLE
Create journal-week-3-reflective.md

### PR DESCRIPTION
**Pull request by kmattiko**

### CODE REVIEW

**Comment by gatorpazz** [comment]
I thought your reflective journal was excellent and is a refreshing change from seeing panic about coding and what now. Actually I think it's pretty darn good writing. You tied in a story about a knitting to our time at the Iron Yard very well. Good start on the informative as well, can't wait to see where it goes. Kudos :+1:

**Comment by sjoyal** [comment]
Great reflective journal Kim. I see all the correlations you're talking about whenever I start in on errands myself. Looking forward to the final product on the informative one as well. :+1:

File: about/html.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+*Tag Structure*
+
+*Non-Word Tag Characters*
```
Comment (al-the-x) [inline about/html.md:3]: One day... One day...

File: about/html.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+`<ul>` and `<ol>` 
+  
+  These are holders for list items. `<ul>s` are unordered lists and <ol>s are ordered lists, that will be numbered. These elements have `<li>s` as children, which will create the actual list. 
+
+`<dl>` 
+
+Stands for Description List. This can be used to store key-value pairs (such as object literals?) or to list a glossary on the page. `<dl>s` are parents to `<dt>s` (used for keys) and `<dd>s` (used for values).
```
Comment (al-the-x) [inline about/html.md:11]: Maybe document `<dt>` and `<dd>`...? (That was the intent of the assignment...)

**Comment by al-the-x** [comment]
Nice work here. You've identified the problem and the solution. Let's see them come together in week 4 and 5.

**Pull request merged**